core-service/src/main/java/com/crmfoodestablishment/coreservice/repository/MenuRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.crmfoodestablishment.coreservice.repository;
+
+import com.crmfoodestablishment.coreservice.entity.Menu;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import java.util.Optional;
+import java.util.UUID;
+
+@Repository
```

Comment:
imo @Repoitory is redundant, since we are implementing JpaRepository

Reply:
ok)